abci/tests/common.rs

```diff
@@ -0,0 +1,161 @@
+use std::time::Duration;
+
+use bollard::{
+    container::{Config, RemoveContainerOptions},
+    service::{CreateImageInfo, HostConfig},
+    Docker, API_DEFAULT_VERSION,
+};
+use futures::StreamExt;
+use tenderdash_abci::Error;
+use tokio::{runtime::Runtime, time::timeout};
+use tracing::{debug, info};
+
+pub struct TenderdashDocker {
+    id: String,
+    docker: Docker,
+    image: String,
+    runtime: Runtime,
+}
+impl TenderdashDocker {
```

Comment:
Did you check that? Because panic by default calls `drop`s until a `drop` itself panics

Reply:
thank you for the testing, if you have time and willingness to improve it you can make `stop` work correctly to cleanup all the things and optionally put it into `Drop`, then make ctrl-c handler consume the value, so it will be dropped automatically

otherwise let's merge it, especially because it's just a test